dist/technology radar - complete.csv

```diff
@@ -21,6 +22,7 @@ Drupal,,1/21/2020,adopt,Platforms,FALSE,"Drupal is an open source content-manage
 Umbraco,,1/11/2020,adopt,Platforms,FALSE,"Umbraco is an open source content management system (CMS) platform written in C# and deployed on Microsoft based infrastructure. <span class=""infoLink"">More info: <a href=""https://our.umbraco.org/documentation/"" target=""_blank"">Click here</a></span>",external,220
 Azure,,1/11/2020,adopt,Platforms,FALSE,Azure is Microsoft's cloud solution.,all,230
 Azure DevOps,,1/11/2020,adopt,Platforms,FALSE,"Azure DevOps provides developer services to support teams to plan work, collaborate on code development, and build and deploy applications.  <span class=""infoLink"">More info: <a href=""https://azure.microsoft.com/en-us/services/devops/?nav=min"" target=""_blank"">Click here</a></span>",all,240
+Bitrise,,2/05/2021,adopt,Platforms,FALSE, Bitrise is a mobile build automation platform <span class=""infoLink"">More info: <a href=""https://www.bitrise.io/"" target=""_blank"">Click here</a></span>.,all,231
```

Comment:
The double quote is missing around the description, not sure if this will break the rendering, but I rather stick to it for consistency at least (before 'Bitrise' and after '</span>.')

Reply:
Fixed